src/encoded/audit/experiment.py

```diff
@@ -1252,18 +1252,25 @@ def check_experiment_long_rna_standards(experiment,
     if 'replication_type' not in experiment:
         return
 
-    mad_metrics = get_metrics(gene_quantifications,
-                              'MadQualityMetric',
-                              desired_assembly,
-                              desired_annotation)
-
     if experiment['assay_term_name'] != 'single-cell RNA sequencing assay':
-        yield from check_spearman(
-            mad_metrics, experiment['replication_type'],
-            0.9, 0.8, pipeline_title)
-    # for failure in check_mad(mad_metrics, experiment['replication_type'],
-    #                         0.2, pipeline_title):
-    #    yield failure
+        mad_metrics = get_metrics(gene_quantifications,
+                                  'MadQualityMetric',
+                                  desired_assembly,
+                                  desired_annotation)
+        replicates = experiment.get('replicates')
+
+        if experiment['replication_type'] == 'unreplicated' and len(replicates) > 1:
+            yield from check_spearman_technical_replicates(
+                mad_metrics, pipeline_title, 0.9)
+
```

Comment:
not a big deal, but I think you can drop the empty lines

Reply:
Got it, I've removed the empty lines in this section